apps/deploy-web/src/components/usage/BillingPage.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { type FC } from "react";
+import { NextSeo } from "next-seo";
+
+import Layout from "@src/components/layout/Layout";
+import { UsageLayout, UsageTabs } from "@src/components/usage/UsageLayout";
+
+export const BillingPage: FC = () => {
```

Comment:
does this need a dedicated feature flag? like if `billing_usage` is enabled user would see this empty page

Reply:
I'm not sure, to me, `billing_usage` ff was only introduced to hide the whole billing usage feature until it's WIP and all related PRs are merged. Of course, I can create another flag for this page specifically, but once this PR is merged, I'll open up another PR with billing transactions list, which are already implemented, for this page at once.

Reply:
Removed tabs layout, Billing page and route for the time being